lib/spack/spack/environment.py

```diff
@@ -826,12 +826,16 @@ def included_config_scopes(self):
                 config_name = 'env:%s:%s' % (
                     self.name, os.path.basename(config_path))
                 scope = spack.config.ConfigScope(config_name, config_path)
-            else:
+            elif os.path.exists(config_path):
                 # files are assumed to be SingleFileScopes
                 base, ext = os.path.splitext(os.path.basename(config_path))
                 config_name = 'env:%s:%s' % (self.name, base)
                 scope = spack.config.SingleFileScope(
                     config_name, config_path, spack.schema.merged.schema)
+            else:
+                tty.warn('Ignoring non-existent include {0}'
+                         .format(config_path))
+                continue
```

Comment:
Given the severity of a missing "include" I wonder if a better approach could be to:
1. Compute all the non-existent files/dirs in this loop
2. Report them after the loop in a `tty.die` error.

Reply:
Done.